Responsividade na Home / FAQ

Leuo29 opened the pull request:
Essa issue é  basicamente:

juntando as issues 81, 75, 74 e 33:

- Trocar Links na home (https://github.com/josircg/saudesraras/issues/81)
- Trocar a palavra noticia por informes (https://github.com/josircg/saudesraras/issues/75)
- Acertar a responsividade na home (https://github.com/josircg/saudesraras/issues/74)
- Ajustar cores em FAQ (https://github.com/josircg/saudesraras/issues/33)

Na issue 75, modifica os seguintes arquivos:

-home
-footer
-navbar
-modalsubmit
-_latest-news


adicionei as alteraçoes abaixo e atualizei o style

Leuo29 committed:
Completando a tarefa 86

A tarefa 86 é uma junçao de 4 tarefas pequenas e todas envolvem a Home:

juntando as issues 81, 75, 74 e 33:

-Trocar Links na home (Trocar Links na home #81)
-Trocar a palavra noticia por informes (trocar a palavra noticia por informes #75)
-Acertar a responsividade na home (acertar a responsividade na home #74)
-Ajustar cores em FAQ (Ajustar cores em FAQ #33)

Na issue 75, modifica os seguintes arquivos:

-home
-footer
-navbar
-modalsubmit
-_latest-news

Leuo29 committed:
Atualizando o styles

Leuo29 committed:
att a traduçao

josircg commented:
Esse branch não está operacional. Está dando erro na home:

django.urls.exceptions.NoReverseMatch: Reverse for 'SUS' not found. 'SUS' is not a valid view function or pattern name.

Leuo29 commented:
Perdao eu tinha colocado o url da pag do SUS que ainda nao teve merge

Leuo29 committed:
resolvendo problema nos links

Esqueci de tirar os links da home que iam pra sus sintomas e diagnosticos, caminhos que ainda nao estao na views

Leuo29 commented:
@josircg consertei

Leuo29 committed:
Update home.html

Leuo29 committed:
Update views.py

Os posts estavam organizados por id e nao por mais recente

josircg commented:
Fiz as atualizações manualmente pois o termo "News" em inglês deve ser mantido. A palavra "informs" é um verbo em inglês.
